.gitignore

```diff
@@ -1,5 +1,3 @@
 ietemplates/
 _site/*
 .idea/
-*.swp
```

Comment:
Probably best leaving that in, it's vim's swap file. People should put it in `.git/info/exclude` if they use vim, but they never do and end up committing tonnes of .swp files.